### ISSUE TITLE
Neofetch ASCII logo looks stretched in Ubuntu Desktop

### ISSUE THREAD

**Issue by elioqoshi** [canonical/Ubuntu-fonts#88]
Original comment: https://discourse.ubuntu.com/t/ubuntu-font-update-available-for-testing/34676/23

![grafik](https://user-images.githubusercontent.com/5436686/232444276-1aa94649-b48b-452c-90c1-718386f92939.png)

The ASCII logo in Neofetch looks distorted with the new font on Ubuntu Desktop. 

I assume this is about the Monospaced font. Is there some variation in line height here?

**Comment by lyubomir-popov** [comment]
hi @djrrb, is this due to some metrics changes to comkply with Google Fonts?

**Comment by djrrb** [comment]
Yes I believe so. This follows their current standards but we can double check that this is the way they want it.

**Comment by lyubomir-popov** [comment]
@seb128 @local-optimum How is this ascii art logo created? If the change is due to a new google fonts standard that we need to adhere to, then maybe we need to regenerate that logo by updating settings in the software that generates it?

**Comment by elioqoshi** [comment]
Not sure if I understand the right technical aspects, but I assume the required change would be to change the line height in the Ubuntu Terminal?

**Comment by lyubomir-popov** [comment]
I doubt we can touch that default. AFAIK ascii art samples an image and then choose the correct character and colour to render that original pixel as a character. What I'm proposing is re-running that generator to sample again, but feed it the updated version of the font, so that it takes the new metrics into account.

Here's an (oversimplified) example. On the left, low line-height text samples  the colour underneath it, and changes colour to the respective colour in the image.

On the right, to simulate the new metrics, I have text with a higher line-height. If every colour samples from the correct (x,y) position from the logo underneath, no stretching would occur.

![image](https://user-images.githubusercontent.com/2741678/232801323-0a2c9a7c-3b3a-48b5-9e84-eff84d189f37.png)

**Comment by joezhouchenye** [comment]
The cell width and height cannot be set to below 1.00 in gnome-terminal unfortunately. I can increase the width to make the shape correct but the text gap may be weird.

Change the Font to Monospace (Noto Mono and some other fonts work well too) and set the cell width to 1.10. The result is good.

The size is changed to 26 rows in my case to display all information without scrolling.

Before:
![image](https://github.com/canonical/Ubuntu-Sans-fonts/assets/59191277/954ac7e0-9777-459d-be2e-b4eb19f0c88f)

After:
![image](https://github.com/canonical/Ubuntu-Sans-fonts/assets/59191277/7d27fdb8-8d78-43a6-97f5-82171904ab68)
